src/index.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+ReactDOM.render(<App />, document.getElementById('root'));
```

Comment:
- [ ] Please make sure the index has 2 `<Link>`s in the navbar which leads to the booklist and categories locations.
The navbar might be in its own component, that's ok!


- [ ] Please make sure the index has 2 `<Route>`s, one for the BookList component and the other for the Categories component. In order to not show a blank page when visiting the site for the first time, the BookList component should be the default page

 


I can see you have implemented this in App.j please move that here for now.